modules/tensor_mechanics/src/postprocessors/CavityPressurePostprocessor.C

```diff
@@ -12,13 +12,11 @@ template<>
 InputParameters validParams<CavityPressurePostprocessor>()
 {
   InputParameters params = validParams<GeneralPostprocessor>();
-  params.addRequiredParam<UserObjectName>("cavity_pressure_uo", "The CavityPressureUserObject that computes the initial moles.");
-  params.addRequiredParam<std::string>("quantity", "The quantity to report.");
-
+  params.addRequiredParam<UserObjectName>("cavity_pressure_uo", "The CavityPressureUserObject that computes the initial moles");
+  params.addRequiredParam<std::string>("quantity", "The quantity to report");
   params.set<bool>("use_displaced_mesh") = true;
 
-  // Hide from input file dump
-  params.addPrivateParam<std::string>("built_by_action", "" );
+  params.addPrivateParam<std::string>("built_by_action", "");// Hide from input file dump
```

Comment:
Good catch! These used to be required before we had the "registerBase()" calls. The purpose was to tie objects to to their base classes for the purpose of putting them under the right syntax sections during dumps and feeding peacock. There is no direct tie from objects to syntax otherwise. The registerBase() methodology is much cleaner and is what we do today.

@hchen139 - just delete this line altogether.